### PR TITLE
Presentation toolbar

### PR DESCRIPTION
**Pull request by Jigar-bhaliya**

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/fullscreen-button_default/styles.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    left :0;
+  }
+
+  [class*="presentationZoomControls"] & {
+    position: relative !important;
+  }
+}
+
+.button {
+
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/fullscreen-button_default/styles.scss]: remove empty line

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader_default/component.jsx [hunk 2]
```diff
@@ -246,21 +246,21 @@ class PresentationUploader extends Component {
...
 
   componentDidUpdate(prevProps) {
-    const { selectedToBeNextCurrent, isOpen, presentations: propPresentations } = this.props;
+    const { isOpen, presentations: propPresentations } = this.props;
     const { presentations } = this.state;
 
     // cleared local presetation state errors and set to presentations available on the server
     if (presentations.length === 0 && propPresentations.length > 1) {
-      return this.setState({ presentations: propPresentations });
+      // return this.setState({ presentations: propPresentations });
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader_default/component.jsx]: do we want this?

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader_default/component.jsx [hunk 3]
```diff
@@ -246,21 +246,21 @@ class PresentationUploader extends Component {
...
     }
 
     // Only presentation available is the default coming from the server.
     // set as selectedToBeNextCurrentOnConfirm once upload / coversion complete
     if (presentations.length === 0 && propPresentations.length === 1) {
       if (propPresentations[0].upload.done && propPresentations[0].conversion.done) {
-        return this.setState({
-          presentations: propPresentations,
-        }, Session.set('selectedToBeNextCurrent', propPresentations[0].id));
+        // return this.setState({
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader_default/component.jsx]: this too?

File: bigbluebutton-html5/imports/ui/components/fullscreen-button_default/styles.scss (hunk 1 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/fullscreen-button_default/styles.scss]: removed

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader_default/component.jsx (hunk 2 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader_default/component.jsx]: fixed btw it's default component and we are not using it.

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader_default/component.jsx (hunk 3 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader_default/component.jsx]: fixed btw it's default component and we are not using it.

**Pull request merged**